src/Analyzers.sln

```diff
@@ -89,6 +89,15 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "RoslynDiagnosticAnalyzersTe
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "RoslynDiagnosticsSetup", "Roslyn\Setup\RoslynDiagnosticsSetup.csproj", "{DEC6D219-A523-422C-AA36-D17662722C0B}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tools", "Tools", "{3FF3C547-451B-4BFF-86D2-FB6A0970D479}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "a2md", "Tools\a2md\a2md.csproj", "{AD31451B-0D31-41A9-8447-66569D2E07B6}"
```

Comment:
What do you think about providing a more descriptive directory name for this tool? e.g. AnalyzerMetadataToMarkdown